packages/client/hmi-client/src/components/model/petrinet/tera-model-diagram.vue

```diff
@@ -214,7 +214,7 @@ watch(
 let graphResizeObserver: ResizeObserver;
 onMounted(() => {
 	if (graphElement.value) {
-		graphResizeObserver = observeElementSizeChange(graphElement.value, renderGraph);
+		graphResizeObserver = observeElementSizeChange(graphElement.value, debounce(renderGraph, 200));
```

Comment:
https://github.com/DARPA-ASKEM/terarium/pull/5174 might be a better fix for the rendering JS-errors.